### PR TITLE
DP-23065 Change header on service page

### PR DESCRIPTION
**Pull request by beto-aveiga**
## Description
- Makes the background images optional.
- When there are no BG images the banner height becomes fluid.
- Update fonts for header and the text below it.
- Adds a variant for banners without image (organisms-page-banner-as-overlay-no-image)
- Other overlay banner variants are not affected.

### organisms-page-banner-as-overlay-no-image (medium)
image

### organisms-page-banner-as-overlay-no-image (large)
image

## Related Issue / Ticket

- [JIRA issue](https://massgov.atlassian.net/browse/DP-23065)

### CODE REVIEW

**Review by arthurbaghdas — APPROVED** [review]
LGTM, I'm a bit unsure what we should do with the failing jobs here. Also I don't have permission to merge into develop. So ideally it will be great if @clairesunstudio can take a quick look and merge it.

File: packages/patternlab/styleguide/source/_patterns/03-organisms/by-template/page-banner~as-overlay-no-image.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "pageBanner": {
+    "size": "overlay",
```
Comment (clairesunstudio) [inline packages/patternlab/styleguide/source/_patterns/03-organisms/by-template/page-banner~as-overlay-no-image.json]: Please update packages/patternlab/styleguide/source/_patterns/03-organisms/by-template/page-banner.md to reflect the changes to this component

**Comment by clairesunstudio** [comment]
I'm seeing some unexpected space caused by an empty image tag when the image goes away on smaller breakpoints 
Screen Shot 2022-01-13 at 2 47 48 PM


On org details page and elected officials page, the image should persist on mobile to my knowledge
Prod: https://mayflower.digital.mass.gov/patternlab/?p=pages-organization-detail
Test: https://mayflower.digital.mass.gov/patternlab/b/DP-23065_Change_header_on_service_page/?p=pages-organization-elected-official

**Comment by clairesunstudio** [comment]
In the visual regression report, I'm seeing some font sizes changes that are unexpected, can you revert the changes for H1? I'm seeing on org pages the H1 fonts has increased the size and on service page H1 decreased in size. H1 should be consistent in sizes. 

Please ignore the size changes for H1 in the design specs, as it's outdated. A ticket has been created separately to address that https://massgov.atlassian.net/browse/DP-23492

**Comment by clairesunstudio** [comment]
Screen Shot 2022-01-18 at 2 37 46 PM

On mobile, the page banner without an image should has solid color background

Screen Shot 2022-01-18 at 2 43 34 PM

In the org details page example, the banner should be configurable whether to render image on mobile. If set to not to render image on mobile, the empty space should be removed
See my approach summary in the ticket --> https://massgov.atlassian.net/browse/DP-23065?focusedCommentId=261155

File: packages/patternlab/styleguide/source/_patterns/03-organisms/by-template/page-banner~as-overlay-no-image.json (hunk 1 above)
Comment (Javi-er) [reply packages/patternlab/styleguide/source/_patterns/03-organisms/by-template/page-banner~as-overlay-no-image.json]: Changes are documented now.

**Comment by clairesunstudio** [comment]
Several issues spotted:
- [x] org page banners don't adjust to the content height - longer title or description has cut off exceeding certain height
- [x] same as elected official banner, and it also has an expected empty space on mobile
- [ ] column variant is broken for existing topic page (to be deprecated)

**Review by clairesunstudio — APPROVED** [review]

**Comment by clairesunstudio** [comment]
linked in https://github.com/massgov/openmass/pull/1109 for external testing purpose

**Review by tom-fleming — APPROVED** [review]
LGTM

**Pull request merged**